pkg/client/core/openapi/zz_generated.openapi.go

```diff
@@ -23,10105 +23,64 @@ limitations under the License.
 package openapi
 
 import (
-	v1alpha1 "github.com/gardener/gardener/pkg/apis/core/v1alpha1"
-	v1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
 	spec "github.com/go-openapi/spec"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	common "k8s.io/kube-openapi/pkg/common"
 )
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return map[string]common.OpenAPIDefinition{
-		"github.com/gardener/gardener/pkg/apis/core/v1alpha1.Addon":                                schema_pkg_apis_core_v1alpha1_Addon(ref),
```

Comment:
Hm, I get these lines readded after `make generate`? Why they are removed?

Reply:
Ran make generate again with same result?

Reply:
@timuthy figured out, that this is related to your installed bash version. Try regenerating with bash version 5.
Seems that `k8s.io/kube-openapi` works only with bash version 5.